doc/Pacemaker_Explained/en-US/Ch-Resources.txt

```diff
@@ -358,7 +358,12 @@ indexterm:[Resource,Option,requires]
 |INFINITY
 |How many failures may occur for this resource on a node, before this
  node is marked ineligible to host this resource. A value of INFINITY
- indicates that this feature is disabled.
+ indicates that this feature is disabled.  Disregarding this option
+ altogether, <<s-failure-migration,the same effect>> has
+ +start-failure-is-fatal+ cluster option set to true (default).
+ This means that for this option, with the exception of INFINITY
+ value (default), to be taken into account, +start-failure-is-fatal+
+ has to be disabled first.
```

Comment:
Not quite true. That's accurate for start failures, but monitor failures will use migration-threshold regardless of the value of start-failure-is-fatal. Also, this option only matters when on-fail=restart for the relevant operation. Maybe:

"This option has an effect only if the failed operation has on-fail=restart (the default), and for failed start operations, if start-failure-is-fatal=false."